frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.unit.spec.tsx

```diff
@@ -156,9 +157,9 @@ describe("Visualizations > PivotTable > PivotTable", () => {
 
     rows.forEach(rowData => {
       columnIndexes.forEach(colIndex => {
-        expect(
-          screen.getByText(rowData[colIndex].toString()),
-        ).toBeInTheDocument();
+        expect(screen.getByTestId("pivot-table")).toHaveTextContent(
```

Comment:
This is effectively doing the same thing that the previous test did except adding the additional row was causing `getBy` to fail because it was finding more than one `baz2`, which was needed for the collapsing test below